flow360/component/simulation/models/material.py

```diff
@@ -22,46 +31,36 @@ class Air(Flow360BaseModel):
     name: Literal["air"] = pd.Field("air", frozen=True)
```

Comment:
Maybe change to `material_name` or `type`? The name entry can leave for user to decide. aluminum.name = "solid" is very weird.